app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/ApplicationController.scala

```diff
@@ -38,7 +38,24 @@ class ApplicationController @Inject() (
     extends CdsController(mcc) {
 
   def startRegister(service: Service): Action[AnyContent] = Action { implicit request =>
-    Ok(viewStartRegister(service))
+    val headingAndTitleText = service.code match {
+      case "gagmr" => "ecc.start-page.title.gagmr"
```

Comment:
Instead of this pattern match you could replace it all with :

s"ecc.start-page.title.$service.code"

This is the same for val bullet2 below